cli/embedded-app/src/views/IframeEmbed.vue

```diff
@@ -9,7 +9,7 @@
     <frame-router
       id="frameRouter"
       v-bind:route="frameRoute"
-      v-on:toastRequest="displayToast"
+      v-on:notifyRequest="displayToast"
```

Comment:
I know it's an internal method, but could rename `displayToast` as well

Reply:
I think I am going to leave these for now. I think another round of search and replace just makes it more likely a typo or error like some of the others y'all caught will slip in, and this is blocking the next release, which has some things other folks are waiting on.